scraper/helper.ts

```diff
@@ -0,0 +1,147 @@
+import { Browser, Page } from 'puppeteer'
+import { WarningTag, ClassWarnings } from './interfaces'
+
+/**
+ * Remove any html character entities from the given string
+ * At this point, it only looks for 3 of them as more are not necessary
+ * @param { string } string The string to remove html characters from
+ * @returns { string }: string with html special characters replaced with english versions of said symbols
+ * @example
+ *    const clean = removeHtmlSpecials('&amp;') // 'and'
+ */
+const removeHtmlSpecials = (str: string) => {
```

Comment:
Transform may be a better name here. Remove implies you're replacing them with nothing, which is not true in this function

```suggestion
const transformHtmlSpecials = (str: string) => {
```

Reply:
Ok. Done